examples/sklearnex/incremental_covariance.py

```diff
@@ -0,0 +1,29 @@
+# ==============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import numpy as np
+
+from sklearnex.covariance import IncrementalEmpiricalCovariance
+
+X = np.array([[0, 1], [0, 1]])
+X_split = np.array_split(X, 2)
+
```

Comment:
Since this will be user facing, I recommend setting up a more realistic use case. For example, you could create 2 variables for the data (perhaps "original_chunk" and "new_chunk") with maybe 3 rows in each (can still keep small 2 columns for simplicity). Run partial fit not in a loop, but on the first chunk of data, maybe a comment describing, and then run partial fit on the second chunk after. Would it make sense to add finalize as well?

Reply:
regarding finalize - it is sklearnex interface which is designed to be like in sklearn where is no call finalize or something similar. finalize is calling when user tries to call any attribute with result data in case if it is necessary to compute these data. also it is calling directly in onedal-like interfaces. (onedal.covariance.IncrementalEmpiricalCovariance)

Reply:
Thanks for clarification. So we can leave any finalize call out of the example, but I would still suggest removing loop and creating 2 separate 3x2 chunks (an original and new) to better illustrate usage.